debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -129,6 +134,7 @@
             .withDisplayName("Oracle version, 11 or 12+")
             .withType(Type.STRING)
             .withImportance(Importance.LOW)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION, 13))
```

Comment:
Since this is deprecated, we aren't showing this in any UI are we?

Reply:
Agree! But then it should also be removed from the ConfigDefinition too? Should @mdrillin remove it from there? @Naros

Reply:
It still needs to be validated, even if it's deprecated.

Reply:
It's not being shown in the UI currently, at least with the way OracleConnectorIntegrator is being used in the UI.